.terraform-version

```diff
@@ -0,0 +1 @@
+1.1.9
```

Comment:
maybe this is annoying? it makes my life nicer.

Reply:
we usually use `asdf` at sourcegraph which uses `.tool-versions`, could that help you, too?